packages/plugin-core/src/showcase/MeetingNotesTip.ts

```diff
@@ -0,0 +1,59 @@
+import { assertUnreachable } from "@dendronhq/common-all";
+import { SegmentClient } from "@dendronhq/common-server";
+import { ShowcaseEntry } from "@dendronhq/engine-server";
+import {
+  MEETING_NOTE_FEATURE_SHOWCASE_TEST,
+  MeetingNoteTestGroups,
+} from "../abTests";
+import { AnalyticsUtils } from "../utils/analytics";
+import {
+  IFeatureShowcaseMessage,
+  DisplayLocation,
+} from "./IFeatureShowcaseMessage";
+
+export class MeetingNotesTip implements IFeatureShowcaseMessage {
+  shouldShow(displayLocation: DisplayLocation): boolean {
+    switch (displayLocation) {
+      case DisplayLocation.InformationMessage: {
+        const ABUserGroup = MEETING_NOTE_FEATURE_SHOWCASE_TEST.getUserGroup(
+          SegmentClient.instance().anonymousId
+        );
+
+        return (
+          ABUserGroup === MeetingNoteTestGroups.show &&
+          !AnalyticsUtils.isFirstWeek()
+        );
+      }
+      case DisplayLocation.TipOfTheDayView:
+        return true;
+      default:
+        assertUnreachable(displayLocation);
+    }
+  }
+  get showcaseEntry(): ShowcaseEntry {
+    return ShowcaseEntry.TryMeetingNotes;
+  }
+
+  getDisplayMessage(displayLocation: DisplayLocation): string {
+    switch (displayLocation) {
+      case DisplayLocation.InformationMessage:
+        return `Dendron now has meeting notes. Try it out!`;
+      case DisplayLocation.TipOfTheDayView:
+        return "Dendron now has meeting notes, which lets you easily create a meeting note with a pre-built template. Try it out now!";
+      default:
+        assertUnreachable(displayLocation);
+    }
+  }
+
+  get onConfirm(): string {
+    return "dendron.createMeetingNote";
+  }
+
+  get confirmText(): string {
+    return "Create a meeting note";
```

Comment:
a second phase but we should keep track of what tips users have clicked on to avoid showing the same tips

Reply:
The Toaster already has this logic.  Tip of the Day doesn't - in other implementations (IntelliJ for example), you can cycle through all the tips and repeat.  It's just that they have a lot of tips, so it'll take many days before you get one you've seen before.